pingouin/parametric.py

```diff
@@ -938,9 +956,9 @@ def anova(data=None, dv=None, between=None, ss_type=2, detailed=False,
     ssbetween = ((grp.mean() - data[dv].mean())**2 * grp.count()).sum()
     # Within effect (= error between)
     #  = (grp.var(ddof=0) * grp.count()).sum()
-    sserror = grp.apply(lambda x: (x - x.mean())**2).sum()
+    error = grp.apply(lambda x: x - x.mean())
```

Comment:
Same here, do you have any documentation / reference implementation for this method of calculating the residuals?